### PR TITLE
Make everything mutable

### PR DESCRIPTION
**Pull request by bpdsw**

### CODE REVIEW

File: src/main/java/org/jarling/v2/models/accounts/Account.java [hunk 1]
```diff
@@ -1,6 +1,9 @@
 package org.jarling.v2.models.accounts;
 
-import lombok.Data;
+import lombok.AllArgsConstructor;
```
Comment (broder) [inline src/main/java/org/jarling/v2/models/accounts/Account.java]: Did you mean to change these imports?

File: src/main/java/org/jarling/v2/models/transactionfeed/CounterPartyType.java [hunk 2]
```diff
@@ -13,7 +13,7 @@
     STARLING("STARLING"),
     LOAN("LOAN");
 
-    private String value;
+    private final String value;
```
Comment (broder) [inline src/main/java/org/jarling/v2/models/transactionfeed/CounterPartyType.java]: I think for enums, using `final` should be okay?

Comment (bpdsw) [reply src/main/java/org/jarling/v2/models/transactionfeed/CounterPartyType.java]: yep, added it in so we're using it on all enums

File: src/main/java/org/jarling/v2/models/accounts/Account.java (hunk 1 above)
Comment (bpdsw) [reply src/main/java/org/jarling/v2/models/accounts/Account.java]: fixed

**Pull request merged**